packages/cli/src/pages/api/lists/[id]/members/[memberId].ts

```diff
@@ -0,0 +1,114 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import {
+  ResError,
+  validate,
+  validationErrorResponse,
+} from "src/util/api/validate";
+import { validateMemberStatusInList } from "src/util/api/validateMemberStatusInList";
+import { withSessionAPIRoute } from "src/util/session";
+import prisma from "../../../../../../prisma";
+
+interface Data {
+  error?: any;
+  member?: any;
+}
+
+async function handlePatchListMember(
+  listId: string,
+  memberId: string,
+  req: NextApiRequest,
+  res: NextApiResponse<Data>
+) {
+  const status = req.body.status;
+
+  const validatedMemberStatusInList = validateMemberStatusInList(status);
+
+  if (validatedMemberStatusInList.hasError) {
+    return validationErrorResponse(validatedMemberStatusInList, res);
+  }
+
+  await prisma.member.update({
+    where: { listId_email: { listId, email: memberId } },
```

Comment:
maybe missed forest for trees.

Is it sketchy to use email as the id here? It allows an attacker to unsub a bunch of people just by knowing their email. If we used a cuid it would remove that problem.

Reply:
Yeah, true, I'll switch it to id.  The unsub link can have the id in the params or look it up from the send id